epyqlib/hildevice.py

```diff
@@ -5,23 +5,18 @@
 import json
 import operator
 import pathlib
-import sys
 import time
 import uuid
 import warnings
-
 import attr
 import canmatrix
-import sunspecdemo.demos
-import sunspecdemo.utils
 import sunspec.core.client
-import twisted.internet.defer
-
 import epyqlib.busproxy
 import epyqlib.canneo
 import epyqlib.device
 import epyqlib.nv
 import epyqlib.utils.qt
+import epyqlib.utils.sunspecdemo_utils
```

Comment:
I agree I don't like the name `sunspecdemo_utils`. Maybe `sunspec_modbus.py` since the file is already under the `utils` directory? I think it is still preferable separate because it is used outside of `hildevice.py` (used by grid-tied albeit in a HIL related file).

Reply:
[d4902cf](https://github.com/epcpower/stlib/pull/48/commits/d4902cf8716663d2901f91bfcb51734bc7f978ff) and [dbff66a](https://github.com/epcpower/stlib/pull/48/commits/dbff66a7607ee90050bab46e60a5732b325cb46f)